src/components/FileInput/FileInput.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ChangeEventHandler, useLayoutEffect, useRef } from "react";
+const fileInput = require("@uswds/uswds/packages/usa-file-input/src");
+
+type IntrinsicElements = JSX.IntrinsicElements["input"];
+
+interface Props extends IntrinsicElements {
+  id: string;
+  name: string;
+  multipleFiles?: boolean;
+  label?: string;
+  acceptedFileTypes?: string[];
+  hintText?: string;
+  disabled?: boolean;
+  error?: boolean;
+  errorMessage?: string;
+  onChange?: ChangeEventHandler<HTMLInputElement>;
+}
+
+/**
+ * **FileInput Component**
+ *
+ * @param {string}    id                  The id to uniquely identify the file input.
+ * @param {string}    name                Name of the file input field.
+ * @param {boolean}   multipleFiles       Controls if the input accepts multiple files.
+ * @param {string}    label               Used to label the file input in the UI.
+ * @param {string[]}  acceptedFileTypes   Array of string file types accepeted in this file input.
+ * @param {string}    hintText            Text to show to the user as a guide of the type of accepateble files.
+ * @param {boolean}   disabled            Controls whether the file input is disabled.
+ * @param {boolean}   error               Triggers error message and error styling.
+ * @param {string}    errorMessage        Message to show when an error is present.
+ * @param {string}    onChange            Function that is called when the File Input changes.
+ *
+ */
+
```

Comment:
Kind of out of scope and control, but it would be really nice to be able to augment these files as well instead of just changing them.
![Screen Shot 2022-10-07 at 12 06 53 PM](https://user-images.githubusercontent.com/22647707/194598642-6160590e-db39-4166-bc46-3d98b2d86e81.png)

Reply:
This feedback would probably more accurately be directed to USWDS.

Reply:
I'm thinking about writing a ticket and just leaving it on the backlog for this project and tackle it later on if it is needed by another team. Don't think I have a lot of control over what is rendered in this component